### PR TITLE
swaybg: split into standalone project

### PR DESCRIPTION
**Pull request by ddevault**
The new upstream is https://github.com/swaywm/swaybg

### CODE REVIEW

**Comment by emersion** [comment]
It seems `swaybar` still needs gdk-pixbuf unfortunately, because of tray icons.

**Comment by ddevault** [comment]
Hm, I'm not sure it actually uses it. It compiles fine without it and that `#ifdef` is the only reference to it I can find.

**Comment by ddevault** [comment]
Nevermind, you're right.

File: common/background-image.c [hunk 1]
```diff
@@ -21,6 +24,103 @@ enum background_mode parse_background_mode(const char *mode) {
 	return BACKGROUND_MODE_INVALID;
 }
 
+#if HAVE_GDK_PIXBUF
+static cairo_surface_t* gdk_cairo_image_surface_create_from_pixbuf(
```
Comment (emersion) [inline common/background-image.c]: Is it intentional to move this from `cairo.c` to this file?

Comment (ddevault) [reply common/background-image.c]: Yeah.

**Comment by emersion** [comment]
Thanks!

**Pull request merged**